pkg/workflows/wasm/host/module.go

```diff
@@ -116,65 +125,60 @@ func NewModule(modCfg *ModuleConfig, binary []byte) (*Module, error) {
 
 	cfg := wasmtime.NewConfig()
 	cfg.SetEpochInterruption(true)
-	cfg.SetConsumeFuel(true)
+	if modCfg.InitialFuel > 0 {
+		cfg.SetConsumeFuel(true)
+	}
 
 	engine := wasmtime.NewEngineWithConfig(cfg)
 
+	if !modCfg.IsUncompressed {
+		rdr := brotli.NewReader(bytes.NewBuffer(binary))
```

Comment:
how did you choose brotli for the compression/decompression?

Reply:
We did some benchmarking between gzip + brotli (and zopfli) and brotli offered the best mix of speed + compression performance.